pkg/virt-handler/vm_test.go

```diff
@@ -834,34 +840,15 @@ var _ = Describe("VirtualMachineInstance", func() {
 				expectEvent(string(v1.AccessCredentialsSyncSuccess), true)
 				updatedVMI, err := virtfakeClient.KubevirtV1().VirtualMachineInstances(metav1.NamespaceDefault).Get(context.TODO(), vmi.Name, metav1.GetOptions{})
 				Expect(err).NotTo(HaveOccurred())
-				Expect(updatedVMI.Status.Conditions).To(ConsistOf(
-					MatchFields(IgnoreExtras, Fields{
-						"Type":   Equal(v1.VirtualMachineInstanceAccessCredentialsSynchronized),
-						"Status": Equal(k8sv1.ConditionTrue)},
-					),
-					MatchFields(IgnoreExtras, Fields{
-						"Type":   Equal(v1.VirtualMachineInstanceIsMigratable),
-						"Status": Equal(k8sv1.ConditionTrue)},
-					),
-					MatchFields(IgnoreExtras, Fields{
-						"Type":   Equal(v1.VirtualMachineInstanceIsStorageLiveMigratable),
-						"Status": Equal(k8sv1.ConditionTrue)},
-					),
-				))
+				Expect(updatedVMI).To(matcher.HaveConditionTrue(v1.VirtualMachineInstanceAccessCredentialsSynchronized))
```

Comment:
What about the other conditions?

Reply:
They are not related to access credentials. I assumed they are irrelevant for this test. Should I return them?